conf/uwsgi.ini

```
@@ -1,5 +1,5 @@
 [uwsgi]
-http=0.0.0.0:8000
+socket = :8000
```

Upon testing, the below seems to have worked on Kubernetes (reverting this line):
```suggestion
http=0.0.0.0:8000
```

---

Didn't need to do this in the end. It was working all along. The application logs (see below) revealed that the application requires jinja templates. These were installed via `npm install`, which had been removed.

```
Traceback (most recent call last):
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django/core/handlers/base.py", line 87, in get_exception_response
fala      |     response = callback(request, **param_dict)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
fala      |     response = view_func(request, *args, **kwargs)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django/views/defaults.py", line 22, in page_not_found
fala      |     body = template.render(context, request)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django_jinja/backend.py", line 64, in render
fala      |     return self.template.render(context)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/jinja2/environment.py", line 1008, in render
fala      |     return self.environment.handle_exception(exc_info, True)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/jinja2/environment.py", line 780, in handle_exception
fala      |     reraise(exc_type, exc_value, tb)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/jinja2/_compat.py", line 37, in reraise
fala      |     raise value.with_traceback(tb)
fala      |   File "/home/app/fala/templates/404.html", line 1, in top-level template code
fala      |     {% extends 'base.html' %}
fala      |   File "/home/app/fala/templates/base.html", line 1, in top-level template code
fala      |     {% extends 'layouts/jinja/external.html' %}
fala      |   File "/home/app/.local/lib/python3.4/site-packages/jinja2/loaders.py", line 187, in get_source
fala      |     raise TemplateNotFound(template)
fala      | jinja2.exceptions.TemplateNotFound: layouts/jinja/external.html
fala      | Traceback (most recent call last):
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django/core/handlers/base.py", line 119, in get_response
fala      |     resolver_match = resolver.resolve(request.path_info)
fala      |   File "/home/app/.local/lib/python3.4/site-packages/django/core/urlresolvers.py", line 388, in resolve
fala      |     raise Resolver404({'tried': tried, 'path': new_path})
fala      | django.core.urlresolvers.Resolver404: {'tried': [[<RegexURLPattern adviser ^$>], [<RegexURLPattern feedback ^feedback/$>], [<RegexURLPattern feedback_confirmation ^feedback/success$>]], 'path': 'favicon.ico'}
```

---

Thanks for checking and clarifying :)